src/gimelstudio/application.py

```diff
@@ -39,12 +38,11 @@ def __init__(self, managed_window):
 
 
 class ApplicationFrame(wx.Frame):
-    def __init__(self):
+    def __init__(self, app_config=None):
```

Comment:
It seems this variable isn't defined. Did you mean to define the ``AppConfiguration`` in ``main.py``?

Reply:
Oops. That was one of the changes I made for making the localisation functional. I can remove this for now (if you wish).

Reply:
That's odd, for me it made the whole dialog to crash with errors. I had cloned your fork to test it out so I would think that I had the latest code. I will try again when the PR is completed. 😄

Reply:
This bug is because of the changes I made in order to get the localisation working. I needed a global ``AppConfiguration`` instance which is why I made the change you commented on (there were changes made in the ``main.py`` file to get this to work). While I will still need these changes implemented for the localisation PR, I will reverse the changes made (in ``main.py`` and the line you made a comment about) for this PR. :slightly_smiling_face: